flow/project.py

```diff
@@ -580,10 +580,45 @@ def __str__(self):
     def __call__(self, *jobs):
         """Return the command formatted with the supplied job(s)."""
         cmd = self._cmd(*jobs) if callable(self._cmd) else self._cmd
-        format_arguments = {"jobs": jobs}
-        if len(jobs) == 1:
-            format_arguments["job"] = jobs[0]
-        return cmd.format(**format_arguments)
+        format_arguments = {}
```

Comment:
It might be nice to add a note about how this function should change once the deprecated behavior is removed. There is a fair bit of `inspect` going on, which I don't think is needed, but I also don't want to fight it if we're going to remove it in the next release.

Reply:
Generally, I agree. I feel though that given @kidrahahjo did the work for this fix we should merge it to have for this version.